src/tree/localProject/LocalProjectTreeItem.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { basename } from "path";
+import { AzExtParentTreeItem, AzExtTreeItem, TreeItemIconPath } from "vscode-azureextensionui";
+import { localize } from "../../utils/localize";
+import { treeUtils } from "../../utils/treeUtils";
+import { ConfigGroupTreeItem } from "./ConfigGroupTreeItem";
+
+export class LocalProjectTreeItem extends AzExtParentTreeItem {
+    public static contextValue: string = 'azureStaticLocalProject';
+    public contextValue: string = LocalProjectTreeItem.contextValue;
+    public readonly label: string = localize('localProject', 'Local Project');
+    public readonly projectPath: string;
+
+    private readonly _projectName: string;
+
+    public constructor(parent: AzExtParentTreeItem, projectPath: string) {
+        super(parent);
+        this.projectPath = projectPath;
+        this._projectName = basename(projectPath);
+    }
+
+    public get iconPath(): TreeItemIconPath {
+        return treeUtils.getThemedIconPath('azure-staticwebapps');
```

Comment:
We should probably look for a different icon to differentiate it from the remote static web apps.  Not this PR, but something to keep in mind.